### PR TITLE
Add readme.md

### PR DESCRIPTION
**Pull request by igorbezsmertnyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,353 @@
-# spree-storefront-api-v2-js-sdk
+# Spree Storefront API SDK
+
+Node module used for integration front-end with Spree API V2
```
Comment (tniezg) [inline README.md]:
```suggestion
Node module for integration with Spree API V2
```

File: README.md [hunk 2]
```diff
@@ -1 +1,637 @@
...
+  `npm install spree-storefront-api-v2-js-sdk --save`
+
+<br/>
+
+2. Initialize `Client`
+
+  ```ts
+    import { Client } from 'spree-storefront-api-v2-js-sdk'
+
+    const client = Client({
```
Comment (tniezg) [inline README.md]: I think with `new` it looks a little more standard:
```suggestion
    const client = new Client({
```

File: README.md [hunk 3]
```diff
@@ -1 +1,637 @@
...
+__failed response schema:__
+
+```ts
+  error: number
+```
+
+__Example:__
+
+```ts
+  import { Client } from 'spree-storefront-api-v2-js-sdk'
```
Comment (tniezg) [inline README.md]: I'd skip writing `import...` and `const client...` in every example.

File: README.md [hunk 4]
```diff
@@ -1 +1,637 @@
...
+
+  const client = Client({
+    host: 'http://localhost:3000'
+  })
+
+  try {
+    const token = await client.authentication.getToken({
```
Comment (tniezg) [inline README.md]: - There's `await` but no `async` visible.
- Can we instead use `then()`?

File: README.md (hunk 2 above)
Comment (igorbezsmertnyi) [reply README.md]: @tniezg `Client` isn't a class

File: README.md (hunk 4 above)
Comment (igorbezsmertnyi) [reply README.md]: I guess we should use new features. I can add `async` function definition if you want. But I think this example shouldn't be detailed, like you described above `I'd skip writing `import...` and `const client...` in every example.` we also cant skip  `async` function

File: README.md (hunk 2 above)
Comment (tniezg) [reply README.md]: I think we should rename the function to `createClient` or `makeClient`. Doesn't have to be in this PR. Uppercase namers are generally reserved for constructors.

File: README.md (hunk 4 above)
Comment (tniezg) [reply README.md]: ok, let's leave as is for now

**Pull request merged**